Makefile

```diff
@@ -4,6 +4,9 @@ BUILDDIR ?= $(CURDIR)/build
 
 BUILD_TAGS?=tendermint
 
+IMAGE := ghcr.io/tendermint/docker-build-proto:latest
+DOCKER_PROTO_BUILDER := docker run -v $(shell pwd):/workspace --workdir /workspace $(IMAGE)
```

Comment:
This variable isn't used.

Reply:
good catch, that has been there for a while, wish we had some bash script linter or something https://github.com/celestiaorg/celestia-core/pull/737/commits/15686cc3e269ff3e82d044d2e25c1d6cc3c6b00d